x-pack/plugins/fleet/public/applications/fleet/app.tsx

```diff
@@ -97,8 +99,11 @@ const PermissionsError: React.FunctionComponent<{ error: string }> = memo(({ err
             <p>
               <FormattedMessage
                 id="xpack.fleet.permissionDeniedErrorMessage"
-                defaultMessage="You are not authorized to access Fleet. Fleet requires {roleName} privileges."
-                values={{ roleName: <EuiCode>superuser</EuiCode> }}
+                defaultMessage="You are not authorized to access Fleet. It requires the {roleName1} Kibana privilege for Fleet, and the {roleName2} or {roleName1} privilege for Integrations."
+                values={{
+                  roleName1: <EuiCode>&quot;All&quot;</EuiCode>,
+                  roleName2: <EuiCode>&quot;Read&quot;</EuiCode>,
+                }}
```

Comment:
I wonder if we still want the code styling and quotes around the "All" and "Read" privilege names. This is how it looks currently, any feedback @mostlyjason @dborodyansky?

![image](https://user-images.githubusercontent.com/1813008/151168037-09745334-2059-4751-977a-d3d8e47da81a.png)